applications/luci-app-udpxy/po/en/udpxy.po

```diff
@@ -1,103 +1,102 @@
 msgid ""
-msgstr ""
```

Comment:
update translation files with `./build/i18n-update.pl applications/luci-app-udpxy/po` or just leave it as-is.